lib/Client.ts

```diff
@@ -83,13 +81,13 @@ export default class Client extends TypedEmitter<ClientEvents> {
 
     /** The client's partial application. This will throw an error if not using a gateway connection or no shard is READY. **/
     get application() {
-        if (!this._application) throw new Error("Cannot access `client.application` before connecting to Discord.");
+        if (!this._application) throw new Error("Can only access `client.application` when using a gateway connection and having one shard marked as READY.");
```

Comment:
```suggestion
        if (!this._application) throw new Error("Cannot access `client.application` without having at least one shard marked as READY.");
```

Maybe some better wording but you get the gist